ios/fastlane/Fastfile

```diff
@@ -2,16 +2,10 @@
 
 require './Constants/Constants'
 require './Constants/Environments'
-require './Managers/BuildManager'
-require './Managers/DistributionManager'
 require './Managers/MatchManager'
 
-builder = BuildManager.new(fastlane: self)
-
-distribution_manager = DistributionManager.new(
-  fastlane: self,
-  build_path: Constants.BUILD_PATH
-)
+FIREBASE_APP_ID = ENV["FIREBASE_APP_ID_STAGING"]
```

Comment:
Here too. Is it from Github secret?

Reply:
It's an environment variable, but the naming is still incorrect so we gotta update it too 🙈